### PR TITLE
Fix user keymaps that aren't compiling

### PR DESCRIPTION
**Pull request by skullydazed**
## Description

A number of user keymaps don't compile. I've been compiling `all:all` a lot lately so having these cleaned up will help me a lot.

## Types of Changes

- [x] Bugfix
- [x] Keymap/layout/userspace (addition or update)

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by skullydazed** [comment]
Pinging @ClashTheBunny @DanielGGordon @magicmonty @martinmksiu @narze @unxmaal @Xyverz @JacobJerrell @jackhumbert @JunoNgx @brandonschlack @callum-oakley @hvpkod 

If your name is above you have one or more files that will be changed by this PR. Please review and let me know if you have any concerns or would rather handle these updates yourself.

File: layouts/community/60_iso/unxmaal/keymap.c [hunk 1]
```diff
@@ -90,37 +90,12 @@ const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
...
         KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS, \
         KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS, \
-        KC_TRNS,F(4),F(5),KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS, \
-        KC_TRNS,KC_TRNS,KC_TRNS,F(6),F(7),F(8),F(9),F(10),F(11),KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,   \
+        KC_TRNS,RGB_TOG,RGB_MOD,KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS, \
+        KC_TRNS,KC_TRNS,KC_TRNS,RGB_HUI,RGB_HUD,RGB_SAI,RGB_SAD,RGB_VAI,RGB_VAD,KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,   \
         KC_TRNS,KC_TRNS,KC_TRNS,          KC_TRNS,               KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS),
 };
 
-enum function_id {
```
Comment (fauxpark) [inline layouts/community/60_iso/unxmaal/keymap.c:98]: 👍 👍 👍

**Comment by JunoNgx** [comment]
I'm cool with the change on my keymap. Thank you very much.

**Review by ClashTheBunny — APPROVED** [review]

**Review by callum-oakley — APPROVED** [review]
fine by me!

File: layouts/community/60_iso/unxmaal/keymap.c (hunk 1 above)
Comment (unxmaal) [reply layouts/community/60_iso/unxmaal/keymap.c:98]: Looks good to me. Thanks!

**Comment by martinmksiu** [comment]
yep happy with it

**Comment by brandonschlack** [comment]
Good with me!

**Review by brandonschlack — APPROVED** [review]

**Review by magicmonty — APPROVED** [review]
I am fine with it. Thanks

**Review by drashna — APPROVED** [review]

**Comment by JacobJerrell** [comment]
Reviewed my files, thanks! Sorry for any mistakes I made/introduced

**Comment by narze** [comment]
Looks like I'm late, but will send you fixes if found :)

**Pull request merged**